src/Microsoft.SqlTools.ServiceLayer/DacFx/Contracts/ValidateStreamingJobParams.cs

```diff
@@ -0,0 +1,44 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+using System.Collections.Generic;
+using Microsoft.SqlTools.Hosting.Protocol.Contracts;
+using Microsoft.SqlTools.ServiceLayer.SchemaCompare.Contracts;
+using Microsoft.SqlTools.ServiceLayer.Utility;
+
+namespace Microsoft.SqlTools.ServiceLayer.DacFx.Contracts
+{
+    /// <summary>
+    /// Parameters for a Validate Streaming Job request.
+    /// </summary>
+    public class ValidateStreamingJobParams
+    {
+        /// <summary>
+        /// Gets or sets the package file path
+        /// </summary>
+        public string PackageFilePath { get; set; }
+
+        /// <summary>
+        /// Gets or sets the create streaming job TSQL.  Should not be used if Statement is set.
+        /// </summary>
+        public string CreateStreamingJobTsql { get; set;}
+    }
+
+    /// <summary>
+    /// Parameters returned from a DacFx get options from profile request.
+    /// </summary>
+    public class ValidateStreamingJobResult : ResultStatus
```

Comment:
Defensive design.  Using the base class that everything else extends from would allow a bug where a different implementation of ResultStatus gets passed back accidentally.  You generally want to head that sort of thing off unless all subclasses are valid/welcome.